cmd/bb_worker/main.go

```diff
@@ -150,12 +150,22 @@ func main() {
 			if err != nil {
 				log.Fatal("Failed to create eviction set for cache directory: ", err)
 			}
+			var linkHelper cas.LinkHelper
+			switch nativeConfiguration.LinkType.LinkType.(type) {
+			case *bb_worker.NativeBuildDirectoryLinkType_Hardlink:
+				linkHelper = cas.NewLinkHelperHardlink()
```

Comment:
For unparameterized types like these, we often follow a pattern where we just provide a singleton instance. See 'InMemoryFilePool' for an example. Could we also use a construct like that over here?